README.md

```diff
@@ -7,7 +7,7 @@ List of ERC-20 tokens in the Hemi Network chains.
 Obtain the latest version of the token list from directly from GitHub:
 
 ```sh
-wget https://github.com/hemilabs/token-list/blob/master/src/hemi.tokenlist.json
+wget https://github.com/hemilabs/token-list/blob/master/src/hemi-sepolia.tokenlist.json
```

Comment:
The idea is to have both the mainnet and the testnet tokens in the same `hemi.tokenlist.json` file. If we rename the file and have two, we would need to import two files everywhere we want to support both chains.

Reply:
ok!, I though we would use different lists, if we'll use only one, then the name is fine, I'll revert it